### DISCUSSION TITLE
Open documentation in Mathematica 13.1

### DISCUSSION THREAD

**Discussion by hongyi18** [FeynCalc/feyncalc#201] [Q&A] [answered]
I wondered if there is a way to open the FeynCalc documentation inside Mathematica, e.g. with the key F1? Since during the installation we were asked to disable the warning of newer versions of Mathematica, I guess the documentation worked well in Mathematica 8 and 9. How about Mathematica 13?

**Comment by Turgon-Aran-Gondolin** [comment, marked as answer]
To my knowledge, you can still open the documentation of a specific function with `F1`, or call out the `FeynCalc` guide by selecting `FeynCalc` and then pressing `F1`. The whole idea of `F1` in Mathematica I think is simply to search for the text you selected in the documentation.

**Reply by vsht** [reply]
As far as FeynCalc 9.3 (aka stable) is concerned, the documentation should actually work as usual.

For the upcoming FeynCalc 10 (current dev version), there will be no documentation integrated into Mma's documentation
center anymore. It is just too much work to maintain and update that...

Instead, there will be (already  is) an online HTML documentation

https://feyncalc.github.io/referenceDev

as well as a PDF manual (same content) that you can download and use offline

https://github.com/FeynCalc/feyncalc-manual/releases/tag/dev-manual

Both are generated from .m files here

https://github.com/FeynCalc/feyncalc/tree/master/FeynCalc/Documentation
